how-we-post - february sales post (#481)
SenseNet/sensenet.github.io: _posts/2021-02-23-how-we-post.md

---
title: "That's how we post"
author: [molnarm84, huszerlbence]
image: "../img/posts/how-we-post.png"
tags: [social media, medium, saas, github, loomly, blog, headless cms]
redirect_to: https://www.sensenet.com/blog/how-we-post
---

> Did you know that this post is born as a .md file and made it's way to our blog with a help of an automated sync tool? Cool, isn't it!?

Reviewer: ```suggestion
Did you know that this post is born as a markdown file and made it's way to our blog with a help of an automated sync tool? Cool, isn't it!?
```

---

We were publishing posts to our blog from time to time but in 2020 we came up with a more strict schedule to regularly post new content about what we are working and what's next.
Now, we have two posts per month published in every two weeks. Since we are working on this as a team, it is important that the document is easily accessible by everyone and a straightforward edit and review process is also a great advantage.

> For that purpose we are using GitHub (like the pros). As a non-developer we struggled a bit at the beginning but once we get used to it, this makes our life easier. Every blogpost (like the source code of sensenet) lives in GitHub in a dedicated repository so we can edit them right from a browser, keep track of every change and also review and comment on it.

Reviewer: ```suggestion
For that purpose we are using GitHub (like the pros). As a non-developer we struggled a bit at the beginning but once we get used to it, this makes our life easier. Every blogpost (like the source code of sensenet) lives on GitHub in a dedicated repository so we can edit them right from the browser, keep track of any changes made and also review and comment it.
```

> When the review is finished and every proposed modifications are made, the PR gets merged into the master branch. The blog itself is part of our public website powered by sensenet so we needed a tool to sync content from GitHub to the sensenet repo behind sensenet.com.

Reviewer: ```suggestion
When the review is finished and every proposed modifications are made, the PR gets merged into the master branch. The blog itself is part of our public website powered by [sensenet](https://sensenet.com/tryit) so we needed a tool to sync content from GitHub to the sensenet repo behind sensenet.com.
```

> When the review is finished and every proposed modifications are made, the PR gets merged into the master branch. The blog itself is part of our public website powered by sensenet so we needed a tool to sync content from GitHub to the sensenet repo behind sensenet.com.

Reviewer: ```suggestion
When the review is finished and all the proposed changes has been made, the PR gets merged into the master branch. The blog itself is part of our public website powered by sensenet so we needed a tool to sync content from GitHub to the sensenet repo behind sensenet.com.
```

> The process is triggered by every merge into the dedicated GitHub repo's master branch then the new post is visible on sensenet.com/blog in all its beauty.

Reviewer: ```suggestion
The process is triggered when a merge is made into the dedicated GitHub repo's master branch then the new post becomes visible on [sensenet.com/blog](https://www.sensenet.com/blog) in all its beauty.
```
